src/test/java/com/amplitude/HttpTransportTest.java

```diff
@@ -260,6 +258,47 @@ public void onLogEventServerResponse(Event event, int status, String message) {
       assertEquals(429, resultMap.get(event));
     }
   }
+  
+  @Test
+  public void testRetryEventWithUserThrottled()
```

Comment:
Thanks for adding this, do we still plan to add a test case for exploring all retries on rate limit?

Reply:
+1